### PR TITLE
[MU4]  Synthesizer routing

### PR DESCRIPTION
**Pull request by igorkorsukov**

### CODE REVIEW

File: framework/midi/internal/fluidlitesynth.cpp [hunk 1]
```diff
@@ -171,42 +171,31 @@ Ret FluidLiteSynth::setupChannels(const Programs& programs)
...
+        fluid_synth_set_interp_method(m_fluid->synth, ch, FLUID_INTERP_DEFAULT);
+        fluid_synth_pitch_wheel_sens(m_fluid->synth, ch, 12);
+    }
 
-        //LOGD() << "ch: " << prog.ch << ", prog: " << prog.prog << ", bank: " << prog.bank << "\n";
+    for (const Event& e: events) {
+        handleEvent(e);
     }
 
     return make_ret(Ret::Code::Ok);
```
Comment (psmokotnin) [inline framework/midi/internal/fluidlitesynth.cpp:192]: make_ret return an instance of Ret, but the function has return type bool

File: framework/midi/internal/zerberussynth.cpp [hunk 2]
```diff
@@ -72,27 +72,24 @@ Ret ZerberusSynth::addSoundFont(const io::path& filePath)
...
 
-    m_programs = programs;
-
-    for (const Program& prog : m_programs) {
-        m_zerb->controller(prog.channel, Ms::CTRL_PROGRAM, prog.program);
+    for (const Event& e : events) {
+        handleEvent(e);
     }
 
     return make_ret(Ret::Code::Ok);
```
Comment (psmokotnin) [inline framework/midi/internal/zerberussynth.cpp:85]: make_ret return an instance of Ret, but the function has return type bool

File: framework/midi/internal/fluidlitesynth.cpp (hunk 1 above)
Comment (igorkorsukov) [reply framework/midi/internal/fluidlitesynth.cpp]: fixed

File: framework/midi/internal/zerberussynth.cpp (hunk 2 above)
Comment (igorkorsukov) [reply framework/midi/internal/zerberussynth.cpp]: fixed

**Pull request merged**